R/tostring.R

```diff
@@ -571,6 +571,7 @@ setMethod("toString", "MatrixPrintForm", function(x,
   # retrieving titles and footers
   allts <- all_titles(x)
 
+  ref_fnotes <- unique(ref_fnotes)
```

Comment:
did you check if this is not messing up pagination indices?

Reply:
Could you add a test for this? It is a bit buggy already be aware

Reply:
Doesn't seem to change the previous ref footnote behaviour with exporters/pagination - I have added another test in rtables just in case.

Reply:
Doing some more testing now